app/src/main/java/com/keylesspalace/tusky/components/drafts/DraftHelper.kt

```diff
@@ -126,6 +127,15 @@ class DraftHelper @Inject constructor(
                 .andThen(draftDao.delete(draft.id))
     }
 
+    fun deleteAllDraftsAndAttachmentsForAccount(accountId: Long) {
+        draftDao.loadDraftsSingle(accountId)
```

Comment:
I would probably rather write SQL for this but I don't think it matters much

Reply:
I'm not sure I understand what you mean? The SQL statement is defined in DraftDao?

Reply:
I mean for the whole deletion thing, sorry for being unclear

Reply:
Ah! But I need to query the drafts so I can delete the attachment files before deleting the drafts in the db. I can't query & delete in 1 statement, can I?

Reply:
I think you can get all those attachments but I think it's a rather complicated query

Reply:
I rather leave it like this then